lib/-ftb-fzf

```diff
@@ -81,7 +82,12 @@ fi
 
 _ftb_query="${_ftb_query}$(command "$dd" bs=1G count=1 status=none iflag=nonblock < /dev/tty 2>/dev/null)" || true
 
-FZF_DEFAULT_OPTS='' SHELL=$ZSH_NAME $fzf_command \
+local fzf_default_opts=''
+if [[ "$use_fzf_default_opts" == "yes" ]]; then
+  fzf_default_opts=$FZF_DEFAULT_OPTS
```

Comment:
I was wondering if `fzf_default_opts` couldn't be set outside like this:
`zstyle ':fzf-tab:*' fzf-default-opts $FZF_DEFAULT_OPTS`
?

I that case you could get ride off the `use_fzf_default_opts` option.

Reply:
Yeah I was going for a quick change so didn't try that. I thought `zstyle` might parse things differently and lead to unintended behaviors.

Let me try that and get back here.

Reply:
Ah I remember it now. Since `FZF_DEFAULT_OPTS` is a bunch of flags, we need to pass it as such and not as an environment variable.

And `fzf-flags` already exists in the plugin. But when I tried setting that equal to the flags in `FZF_DEFAULT_OPTS` (see below), it wasn't working. I think the reason is the same as what I mentioned in https://github.com/Aloxaf/fzf-tab/issues/475#issuecomment-2395474675.

```sh
# this converts the string to a string array which is expected by fzf-flags.
export parsed_fzf_default_opts=(${=FZF_DEFAULT_OPTS})

zstyle ':fzf-tab:*' fzf-flags $parsed_fzf_default_opts
```

Reply:
Your suggestion doesn't seem to be working for my case:
```
export FZF_DEFAULT_OPTS="          \
  --exact                          \
  --height ${FZF_TMUX_HEIGHT:-30%} \
  --layout=reverse                 \
  --info=hidden                    \
  --prompt='$  '                   \
  --pointer='=>'                   \
  --marker='+'                     \
  --cycle                          \
  --select-1                       \
  --color=dark                     \
  --bind '?:toggle-preview,ctrl-a:select-all,ctrl-d:half-page-down,ctrl-u:half-page-up,ctrl-l:clear-query,home:first,end:last,enter:accept-non-empty,alt-down:down,alt-up:up,right:accept-non-empty,alt-right:accept-non-empty,alt-left:close,left:close' \
  --color='fg:-1,bg:-1,hl:#fb8aa4,fg+:-1,bg+:-1,hl+:#55E579' \
  --color='info:#af87ff,prompt:#fb8aa4,pointer:#55E579,marker:#55E579,spinner:#55E579'
"
```

I think the `--height ${FZF_TMUX_HEIGHT:-30%}` is not correctly handle.

Reply:
@pablospe can you try this?

```zsh
zstyle ':fzf-tab:*' fzf-flags ${(Q)${(Z:n:)FZF_DEFAULT_OPTS}}
```

Reply:
This is magic! It is working, thanks!

Reply:
These are my flags,
```
# Set fzf options
export FZF_DEFAULT_OPTS=$FZF_DEFAULT_OPTS'
    --color=bg:#1b1b1b
    --color=preview-bg:#1b1b1b
    --color=hl+:#fb4934
    --color=gutter:#1b1b1b
    --color=pointer:#f9f5d7,marker:#f9f5d7
    --color=info:#ebdbb2,spinner:#f9f5d7
    --color=query:#ebdbb2,prompt:#ebdbb2
    --color=border:#ebdbb2
    --border="rounded" --prompt="> "
    --marker="" --pointer="->" --layout="reverse"
'
```

While `echo ${(Q)${(Z:nC:)FZF_DEFAULT_OPTS}}` looks as expected, when I try to open fzf-tab, it fails with an eval error.